static-analysis/teammates-pmd.xml

```diff
@@ -117,15 +117,15 @@
    <!--<rule ref="rulesets/java/design.xml/UseUtilityClass"/>-->
    <!--<rule ref="rulesets/java/design.xml/UseVarargs"/>-->
    <!--<rule ref="rulesets/java/basic.xml/EmptyCatchBlock"/>-->
-   <!--<rule ref="rulesets/java/basic.xml/EmptyFinallyBlock"/>-->
-   <!--<rule ref="rulesets/java/basic.xml/EmptyInitializer"/>-->
-   <!--<rule ref="rulesets/java/basic.xml/EmptyStatementBlock"/>-->
-   <!--<rule ref="rulesets/java/basic.xml/EmptyStatementNotInLoop"/>-->
-   <!--<rule ref="rulesets/java/basic.xml/EmptyStaticInitializer"/>-->
-   <!--<rule ref="rulesets/java/basic.xml/EmptySwitchStatements"/>-->
-   <!--<rule ref="rulesets/java/basic.xml/EmptySynchronizedBlock"/>-->
-   <!--<rule ref="rulesets/java/basic.xml/EmptyTryBlock"/>-->
-   <!--<rule ref="rulesets/java/basic.xml/EmptyWhileStmt"/>-->
+   <rule ref="rulesets/java/basic.xml/EmptyFinallyBlock"/>
+   <rule ref="rulesets/java/basic.xml/EmptyInitializer"/>
+   <rule ref="rulesets/java/basic.xml/EmptyStatementBlock"/>
+   <rule ref="rulesets/java/basic.xml/EmptyStatementNotInLoop"/>
+   <rule ref="rulesets/java/basic.xml/EmptyStaticInitializer"/>
+   <rule ref="rulesets/java/basic.xml/EmptySwitchStatements"/>
+   <rule ref="rulesets/java/basic.xml/EmptySynchronizedBlock"/>
+   <rule ref="rulesets/java/basic.xml/EmptyTryBlock"/>
+   <rule ref="rulesets/java/basic.xml/EmptyWhileStmt"/>
```

Comment:
Apparently they've changed the package name to `empty.xml`, so this stopped the build from proceeding previously.